src/data/actions/assets.test.jsx

```diff
@@ -138,6 +149,42 @@ describe('Assets Action Creators', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+  it('returns expected state from toggleLockAsset success', () => {
```

Comment:
Farhanah originally reviewed the two toggleLockAsset tests below, but I accidentally removed them when rebasing my previous test branch 😞 . I've tried to reproduce them as closely to the originals as I could remember. I think that they're probably fine to not be reviewed, since they've been looked at, but feel free to take a look if you want to.